src/MongoDB.Bson/Serialization/Serializers/DiscriminatedInterfaceSerializer.cs

```diff
@@ -110,5 +110,25 @@ public override void Serialize(BsonSerializationContext context, BsonSerializati
                 _objectSerializer.Serialize(context, args, value);
             }
         }
+
+        /// <inheritdoc />
+        public bool TryGetMemberSerializationInfo(string memberName, out BsonSerializationInfo serializationInfo)
+        {
+            var memberType = _interfaceType.GetTypeInfo().GetProperty(memberName);
```

Comment:
Need flags to restrict to public instance properties?

Reply:
Yes, sounds good. I forget about it.